packages/cfd/src/Components/cfd-real-account-display.tsx

```diff
@@ -200,7 +200,7 @@ const CFDRealAccountDisplay = ({
             platform={platform}
             descriptor={
                 platform === 'mt5'
-                    ? localize('Trade CFDs on our synthetics and basket indices.')
```

Comment:
This is only changes the Real account text. you need to implement it for Demo as well.

Reply:
Demo was changed already.